api.planx.uk/lib/notify/templates/saveAndReturn/user-confirmation.ts

```diff
@@ -0,0 +1,20 @@
+import { type NotifyConfig, type NotifyTemplate } from "../index.js";
+
+/**
+ * GovNotify template: https://www.notifications.service.gov.uk/services/012e65af-0eb2-45d5-87bd-4248354c4c22/templates/8b82b606-defa-4daa-8fdb-e78b852b8ffb
+ */
+export const userConfirmationTemplate: NotifyTemplate<Config> = {
+  id: "8b82b606-defa-4daa-8fdb-e78b852b8ffb",
+  access: "private",
+  config: {} as Config,
+};
+
+export type Config = NotifyConfig<{
+  serviceName: string;
+  sessionId: string;
+  address: string;
+  projectType: string;
+  helpEmail: string;
+  helpOpeningHours: string;
+  helpPhone: string;
+}>;
```

Comment:
Agree one-file-per-template with repeated types feels like a much easier-to-work-with solution here than previous types, the overlap is tedious!

Does feel like there's one small type that's maybe useful to extract though - something like: 
```ts
type EmailFooter = {
  helpEmail: string;
  helpOpeningHours: string;
  helpPhone: string;
}; 

export type Config = NotifyConfig<{EmailFooter & {...}>;
``` 

Just a thought ! Feels slightly useful in my mental modal to separate fields related to applicant journey and team-level settings if possible. Aware not even the footer is share among all templates though !

Reply:
Great shout 🙂  I'll add this now.